src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -202,7 +208,7 @@ const styles = StyleSheet.create({
   },
   container: {
     backgroundColor: COLORS.default_signed_secondary,
-    borderRadius: 15,
+    borderRadius: 16,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the borderRadius from 15 to 16 is a minor change and should not affect the overall look and feel of the container. However, it's always a good practice to maintain consistent styling across the application. If 16 is the standard value used elsewhere in the app, then this change is fine.
```diff
- borderRadius: 16,
+ borderRadius: 15,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->